src/DOM/HTML/Window.purs

```diff
@@ -102,3 +109,31 @@ foreign import scrollY :: forall eff. Window -> Eff (dom :: DOM | eff) Int
 foreign import localStorage :: forall eff. Window -> Eff (dom :: DOM | eff) Storage
 
 foreign import sessionStorage :: forall eff. Window -> Eff (dom :: DOM | eff) Storage
+
+newtype RequestAnimationFrameId = RequestAnimationFrameId Int
+
+derive instance newtypeRequestAnimationFrameId :: Newtype RequestAnimationFrameId _
```

Comment:
Can you derive `Eq` and `Ord` for both of the newtypes introduced in here please?